src/chi/components/header/header.scss

```diff
@@ -95,16 +95,9 @@
       flex-direction: row;
       height: 2rem;
       justify-content: center;
+      padding: 0 1rem;
       position: relative;
       width: 100%;
-      @include respond-to(sm) {
```

Comment:
@davidojedacl please return this piece of code, as we saw it was in the original header version

Reply:
done